tests/envs/test_blocks.py

```diff
@@ -40,12 +38,13 @@ def test_blocks():
                 assert block.type.name == "robot"
                 continue
             assert not (state.get(block, "held") and state.get(block, "clear"))
-        act = Action(env.action_space.sample())
         if i == 0:
-            with pytest.raises(NotImplementedError):
-                env.render(state, task, act)
-        env.simulate(state, act)
-
+            # Force initial pick to test rendering with holding
+            block = sorted([o for o in state if o.type.name == "block" and \
+                            state.get(o, 'clear') > env.clear_tol])[0]
+            pick = env._Pick_policy(state, [block], np.zeros(3))  # pylint: disable=protected-access
```

Comment:
instead of this protected access, can we do it the way i did in the function below?
```
Pick = [o for o in env.options if o.name == "Pick"][0]
act = Pick.ground([block1], np.array(
        [0, 0, 0], dtype=np.float32)).policy(state)
```